### PR TITLE
[13.0][ADD] sale_stock: Nothing to do

### PR DESCRIPTION
**Pull request by pedrobaeza**
@Tecnativa TT21675

### CODE REVIEW

File: addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---Models in module 'sale_stock'---
+---Fields in module 'sale_stock'---
+sale_stock   / account.invoice          / incoterms_id (many2one)       : DEL relation: account.incoterms
+# NOTHING TO DO: This field was duplicated in the v12 DB schema, and it was hidden in views, so no possible values in it
```
Comment (kos94ok-3D) [inline addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt:4]: Now a model `account.move` and an `invoice_incoterm_id` field so was done in an `account` module

Comment (pedrobaeza) [reply addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt:4]: Not really. That's another field. An upcoming v12 patch will "merge" the values of both fields where it belongs.

Comment (pedrobaeza) [reply addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt:4]: See #2544

Comment (kos94ok-3D) [reply addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt:4]: Oh, I see. Thanks.

**Review by kos94ok-3D — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Again I have forgotten about the dependency... but I'm reviewing it.

**Pull request merged**